src/quo2/components/common/notification_dot/view.cljs

```diff
@@ -0,0 +1,11 @@
+(ns quo2.components.common.notification-dot.view
+  (:require [react-native.core :as rn]
+            [quo2.components.common.notification-dot.style :as style]))
+
+(defn notification-dot
```

Comment:
nit we can just wrap this with the `with-theme` HOC so we don't have to pass `theme` as a prop 👍